app/models/user.rb

```diff
@@ -1,6 +1,29 @@
 class User < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable and :omniauthable
+  enum status: [:disable, :enable]
+  enum role: [:user, :admin, :manager]
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
+  scope :notadmin, -> { where.not(role: :admin) }
+
+  def active_for_authentication?
+    super && enable?
+  end
+  
+  def inactive_message
+    if enable?
+       super
```

Comment:
indentation

Reply:
ternary operator can be used here, as you have used it below in 2 more similar cases.

Reply:
changed.